model/tools/ftn2src.sh

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Do we want a nice shebang at the top? E.g.
`!/bin/bash`

Reply:
Smart idea! I'll add this.